### PR TITLE
Splits render time into separate column

### PR DESCRIPTION
**Pull request by locks**
Ember Inspector - Render Performance

- [x] Separate render time into column
- [x] ~~Make render time column sortable~~
- [x] Add odd/even row style

### CODE REVIEW

File: app/templates/components/render-item.hbs [hunk 1]
```diff
@@ -3,10 +3,12 @@
     <span class="list__cell-arrow"></span>
     <span title="{{model.name}}">
       <span class="js-render-profile-name">{{model.name}}</span>
-      <span class="pill pill_not-clickable js-render-profile-duration">{{ms-to-time model.duration}}</span>
     </span>
   {{/list.cell}}
   {{#list.cell class="list__cell_value_numeric js-render-profile-timestamp"}}
```
Comment (teddyzeenny) [inline app/templates/components/render-item.hbs]: `js-render-profile-timestamp` does not apply to this column anymore. Maybe `js-render-profile-duration`.

**Comment by teddyzeenny** [comment]
👍

**Comment by teddyzeenny** [comment]
> Make render time column sortable

This is a bit tricky since it's a hierarchical list so I don't think you can completely sort since you'd need to preserve the hierarchy.

**Comment by locks** [comment]
> This is a bit tricky since it's a hierarchical list so I don't think you can completely sort since you'd need to preserve the hierarchy.
That's a good point, removing it.

**Pull request merged**